io.github.martchus.syncthingtray.yml

```diff
@@ -161,6 +159,7 @@ modules:
       - -DNO_PLASMOID=ON
       - -DNO_FILE_ITEM_ACTION_PLUGIN=ON
       - -DNO_AUTOSTART_SETTINGS:BOOL=ON
+      - -DMETA_DEVELOPER_ID:STRING=io.github.martchus
```

Comment:
This should now be the default. So I suggest you remove that option because it is really meant for testing/development and to ease changing it in the future (in case it is necessary for whatever reason).

Reply:
sure

Reply:
Thanks, I suppose https://github.com/flathub/io.github.martchus.syncthingtray/pull/2 can be closed then.

Reply:
yeah, I just leave it there to check if the bot can close it automatically.

I assume the bot send two PR because we use [Anitya](https://github.com/fedora-infra/anitya) service to check for syncthing update, which may have some delay compare to official release page.

If that is the case, after Syncthing Tray has new release, we may need to wait for enough time (e.g. at least 3 hours as this two PR time interval) to let the service detect dependency update.